LayoutTests/fast/webgpu/nocrash/RenderBundle_WRITE.html

```diff
@@ -49,3 +51,5 @@
     globalThis.testRunner?.notifyDone();
   };
 </script>
+Pass
```

Comment:
> could you still add simple message like "Passed." upon successfully executing the test?

This message is not added after executing the test.. It's there already in the document, whether or not the test executes..

Reply:
Added a console.log statement right before notify done now